src/main.rs

```diff
@@ -34,7 +34,7 @@ fn main() -> Result<(), Error> {
         .get_matches();
 
     let client = HttpClient::new(
-        reqwest::Client::new(),
+        reqwest::blocking::Client::new(),
```

Comment:
IIUC, don't we need to conditionally choose the reqwest client based on whether or not the `blocking` feature is enabled?

Reply:
The `HttpClient` requires a blocking reqwest client (the APIs aren't compatible - one is standard functions and the other is async functions).

A couple notes:
1. We don't actually use the client from this crate - we use our own client that uses some of the other helper code (enums and structs).
2. This command line tool is for manually sending requests to Segment; we don't use it but I just happened to notice (when running tests) that it wasn't compiling so I fixed it here.  (I don't think anyone on our team has actually used this.)

Reply:
Gotcha, thanks for the context